modules/brew/README.md

```diff
@@ -0,0 +1,136 @@
+  - `/var/lib/homebrew/`
+  - `/var/cache/homebrew/`
+  - `/home/linuxbrew/`
+- `brew-setup` service is enabled
+
+### Boot-time:
+
+**`tmpfiles.d homebrew.conf`:**
+- This configuration is telling SystemD to: automatically create these necessary directories on every system boot if not available & to give them permissions of the default user (UID 1000):
+  - `/var/lib/homebrew/`
+  - `/var/cache/homebrew/`
+  - `/home/linuxbrew/`
+
+**`brew-setup`:**
+- `brew-setup` SystemD service checks if main directory used by Brew exists (`/home/linuxbrew/.linuxbrew/`)  
+  & if `brew-setup` state file exists (`/etc/.linuxbrew`)
+- If one of those paths don't exist, than Homebrew tar is extracted from `/usr/share/homebrew/homebrew.tar.zst` to `/tmp/homebrew/`
+- Extracted Homebrew is than copied from `/tmp/homebrew/` to `/home/linuxbrew/` & permissions to it are set to default user (UID 1000)
+- Temporary directory `/tmp/homebrew/` is removed
+- Empty file `/etc/.linuxbrew` is created, which indicates that brew-setup (installation) is successful & which allows setup to run again on next boot when removed
+
+**Rest of the setup:**
+- `brew-update` is runnning in specified time to update Brew binary to latest version
+- `brew-upgrade` is runnning in specified time to upgrade Brew packages to latest versions
+
+## Configuration Options
+
+### `update-interval` (optional: string, default: '6h')
+Defines how often the Brew update service should run. The string is passed directly to `OnUnitInactiveSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `upgrade-interval` (optional: string, default: '8h')
+Defines how often the Brew upgrade service should run. The string is passed directly to `OnUnitInactiveSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `wait-after-boot-update` (optional: string, default: '10min')
+Time delay after system boot before the first Brew update runs. The string is passed directly to `OnBootSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `wait-after-boot-upgrade` (optional: string, default: '30min')
+Time delay after system boot before the first Brew package upgrade runs. The string is passed directly to `OnBootSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `auto-update` (optional: boolean, default: true)
+If false, disables automatic activation of `brew-update.timer`.
+
+### `auto-upgrade` (optional: boolean, default: true)
+If false, disables automatic activation of `brew-upgrade.timer`.
+
+!!! warning
+    Please review the brew documentation carefully before modifying these settings.
+
+### `nofile-limits` (optional: boolean, default: false)
+Determines whether to apply nofile limits for Brew installations. When set to true, it increases the nofile limits to prevent certain packages from failing due to file limits. However, it's important to note that increasing nofile limits can have potential security implications. Defaults to false for security purposes.
```

Comment:
I have no idea what `nofile limits` and what that would mean for me. I'd also be interested in knowing if these limits are set by default when installing brew the official way. I googled the term, but did not find any information about security implications. 

I also assume that the warning above applies to this setting mostly, so a link to the referenced Brew documentation would be great, if that exists.

Reply:
It is not set by default in `brew`.

Homebrew doesn't have documentation about this either, nor they are not aware about this afaik.

It is something that Bluefin maintainers noticed, where some packages installed from `brew` complain about reached nofile limits & refuse to work due to this.
https://github.com/ublue-os/bluefin/pull/988

This doesn't just happen with `brew` packages in particular, it can happen with other packages too.
Why this happens in more detail, I have no idea.

About security implications, this is just my own view of it. It is indeed true that there are no searchable topics regarding security of nofile limits.

Maybe @qoijjj knows something more about this.